### PR TITLE
Bugfix/component disappear when select first

### PR DESCRIPTION
**Pull request by Hundraw**
Fixed the bug that made the plugin core crash when reloading with a selected component not yet displayed

### CODE REVIEW

File: cypress/e2e/renderer.feature [hunk 1]
```diff
@@ -42,3 +42,45 @@ Feature: Test plugin renderer
...
+  And I expect "#workflow1" is visible
+  And I expect "#wfstep4" is visible
+  And I expect "#wfstep3" is visible
+  And I expect "#wfstep2" is visible
+  And I expect "#wfstep1" is visible
+  And I expect "#workflow2" is visible
+  And I expect "#wfstep8" is visible
+  And I expect "#wfstep7" is visible
+  And I expect "#wfstep6" is visible
+  And I expect "#wfstep5" is visible
```
Comment (justinehell) [inline cypress/e2e/renderer.feature:86]: Add line at end of file

File: cypress/e2e/selection.feature [hunk 2]
```diff
@@ -68,4 +68,4 @@ Feature: Test component/link selection
     And I expect component "#link-server-laptop_link-server1-external1" to have style "outlineStyle" set to "solid"
     And I expect component "#link-server-laptop_link-server1-external1" to have style "outlineWidth" set to "2px"
     And I expect "#action-menu" is visible
-    And I expect "#remove-link" is visible
+    And I expect "#remove-link" is visible
```
Comment (justinehell) [inline cypress/e2e/selection.feature:71]: Add line at end of file

File: demo/src/DemoPlugin.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { DefaultData, DefaultPlugin } from 'leto-modelizer-plugin-core';
+import { DefaultData, DefaultPlugin, Component } from 'leto-modelizer-plugin-core';
```
Comment (justinehell) [inline demo/src/DemoPlugin.js:1]: `Component` not used

**Review by justinehell — CHANGES_REQUESTED** [review]

File: demo/src/App.vue [hunk 4]
```diff
@@ -14,7 +15,7 @@
 import { onMounted } from 'vue';
 import resources from './assets/resources';
 import DemoPlugin from '@/DemoPlugin';
-import { ComponentDrawOption, FileInput } from 'leto-modelizer-plugin-core';
+import { Component, ComponentDrawOption, FileInput } from 'leto-modelizer-plugin-core';
```
Comment (k-y0u) [inline demo/src/App.vue:18]: divide (3 or more)

File: demo/src/DemoPlugin.js (hunk 3 above)
Comment (k-y0u) [inline demo/src/DemoPlugin.js:1]: divide

File: cypress/e2e/renderer.feature [hunk 5]
```diff
@@ -42,3 +42,45 @@ Feature: Test plugin renderer
     And I expect "#link-__workflow-__next-wfstep5-wfstep6" is visible
     And I expect "#link-__workflow-__next-wfstep6-wfstep7" is visible
     And I expect "#link-__workflow-__next-wfstep7-wfstep8" is visible
+
+Scenario: Reset components with a selected component that does not exist
+  Then I expect ".component" appear 16 times on screen
+  And I expect ".component-DefaultModel" appear 12 times on screen
```
Comment (k-y0u) [inline cypress/e2e/renderer.feature:48]: align I

**Review by k-y0u — DISMISSED** [review]

File: cypress/e2e/renderer.feature (hunk 1 above)
Comment (Hundraw) [reply cypress/e2e/renderer.feature:86]: Done

File: cypress/e2e/selection.feature (hunk 2 above)
Comment (Hundraw) [reply cypress/e2e/selection.feature:71]: Done

File: demo/src/DemoPlugin.js (hunk 3 above)
Comment (Hundraw) [reply demo/src/DemoPlugin.js:1]: Done

File: demo/src/App.vue (hunk 4 above)
Comment (Hundraw) [reply demo/src/App.vue:18]: Done

File: cypress/e2e/renderer.feature (hunk 5 above)
Comment (Zorin95670) [reply cypress/e2e/renderer.feature:48]: Not done, but link to a linter improvement.

File: cypress/e2e/renderer.feature [hunk 6]
```diff
@@ -42,3 +42,45 @@ Feature: Test plugin renderer
...
+  And I expect "#wfstep1" is visible
+  And I expect "#workflow2" is visible
+  And I expect "#wfstep8" is visible
+  And I expect "#wfstep7" is visible
+  And I expect "#wfstep6" is visible
+  And I expect "#wfstep5" is visible
+
+  When I click on '.reset-btn'
+  Then I expect ".component" appear 16 times on screen
+  And I expect ".component-DefaultModel" appear 12 times on screen
```
[the remark this answers is not in the document]
Comment (Zorin95670) [reply cypress/e2e/renderer.feature:70]: Not done, but link to a linter improvement.

File: demo/src/App.vue (hunk 4 above)
Comment (Zorin95670) [reply demo/src/App.vue:18]: Not done, because is not a linter error.

**Review by Zorin95670 — APPROVED** [review]

File: demo/src/DemoPlugin.js (hunk 3 above)
Comment (Zorin95670) [reply demo/src/DemoPlugin.js]: Not done because is not a linter error

**Pull request merged**